lib/hex_web/owners.ex

```diff
@@ -0,0 +1,62 @@
+defmodule HexWeb.Owners do
```

Comment:
You have good point that there will be less "CRUD" modules than schema modules. So I think `HexWeb.{Docs, Packages, Releases, Users}` could be fine as names. Do you know a good name for the directory they should be in?

There's a repository where this being discussed for Phoenix. I pinged Chris about giving you access. Ping me on IRC if you want to discuss it further.